cmd/scaffold.go

```diff
@@ -1113,6 +1113,8 @@ func (fnb *FlowNodeBuilder) RegisterDefaultAdminCommands() {
 		return storageCommands.NewReadResultsCommand(config.State, config.Storage.Results)
 	}).AdminCommand("read-seals", func(config *NodeConfig) commands.AdminCommand {
 		return storageCommands.NewReadSealsCommand(config.State, config.Storage.Seals, config.Storage.Index)
+	}).AdminCommand("get-identity", func(config *NodeConfig) commands.AdminCommand {
+		return common.NewGetIdentityCommand(config.IdentityProvider)
```

Comment:
It would be great to add example `curl` command lines to the `README.md` [1] file like the other commands :-)
[1] https://github.com/onflow/flow-go/tree/cd1449feabdaa2be1ebe1c5ccb51eb96f1649b80/admin